packages/cli/src/cmds/validator/keymanager/interface.ts

```diff
@@ -26,4 +27,9 @@ export interface IPersistedKeysBackend {
 
   /** Returns true some item is deleted from disk */
   deleteRemoteKey(pubkey: PubkeyHex): boolean;
+
+  writeProposerConfig(pubkey: PubkeyHex, proposerConfig: ProposerConfig | null): void;
+  deleteProposerConfig(pubkeyHex: PubkeyHex): void;
+  readAllProposerConfigs(): {[index: string]: ProposerConfig};
```

Comment:
I don't think the `All` part of the name is necessary. The plural at the end conveys enough meaning

Reply:
sounds good, should i also change `readAllKeystores` to `readKeystores` from the previous work?

Reply:
Oh sorry brain fart, though this was the public API. naming is good :+1: